lib/lita/adapters/slack/user_creator.rb

```diff
@@ -11,6 +11,7 @@ def create_user(slack_user, robot, robot_id)
             )
 
             update_robot(robot, slack_user) if slack_user.id == robot_id
+            robot.trigger(:slack_user_created, user_id: slack_user.id, raw_data: slack_user.raw_data)
```

Comment:
Since all the data is being stored inside the `SlackUser` object now, I think it'd make more sense to simply pass that object in the event payload rather than pulling off the ID and raw data as two separate things. Then plugins listening to the event can extract whatever information they want from the `SlackUser`.